mediapipe/tasks/ios/test/audio/core/utils/sources/AVAudioFile+TestUtils.m

```diff
@@ -63,9 +63,10 @@ @implementation AVAudioFile (TestUtils)
     MPPAudioData *audioData =
         [[MPPAudioData alloc] initWithFormat:audioDataFormat
                                  sampleCount:lengthToBeLoaded / audioDataFormat.channelCount];
-
+    // Can safely access `floatChannelData[0]` since the input file is expected to have atleast 1
+    // channel.
     MPPFloatBuffer *floatBuffer =
-        [[MPPFloatBuffer alloc] initWithData:audioPCMBuffer.floatChannelData[currentPosition]
+        [[MPPFloatBuffer alloc] initWithData:audioPCMBuffer.floatChannelData[0] + currentPosition
```

Comment:
Fingers crossed that this compiles internally (the original code did not...)

Reply:
This is a runtime issue. Just returns garbage values.

I had the same problem I think. You may have used the default --config or --config=darwin_arm64 or darwin_x86_64 when building it. You can use --config=ios --cpu= darwin_x86_64 or darwin_arm64.
Also, the tests will run just fine with whatever config you usually for them.

It's because this one depends on `MPPAudioData` which transitively depends on `AVAudioSession`. `AVAudioSession` is not available on macOS. So you have to use one of the iOS configs to compile it.